examples/inverse/plot_custom_inverse_solver.py

```diff
@@ -40,8 +40,8 @@
 
 evoked = evoked.pick_types(eeg=False, meg=True)
 # Handling forward solution
-forward = mne.read_forward_solution(fwd_fname, surf_ori=True)
-
+forward = mne.read_forward_solution(fwd_fname)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
What about examples? Shall I use `use_cps=False` if fixed forwards are used to make sure that the results won't change when we change the default. Or is it ok to use `use_cps=True` and potentially change the results in order to use the default in the examples.